setup.py

```diff
@@ -37,4 +37,7 @@
         ],
     },
     packages=find_packages(exclude=('tests*', 'testing*')),
+    package_data={
+        'sticht': ['py.typed'],
+    },
```

Comment:
hmm, i guess i should add this in my mypy branch - but this is a pretty innocuous change (just lets consumers use our type annotations when they dep us)